### PR TITLE
Show number of days

### PR DESCRIPTION
**Pull request by timjkong**
Ui now shows number of days for accommodations and travel plans

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -21,5 +21,7 @@
     public static final String MESSAGE_ACCOMMODATIONS_LISTED_OVERVIEW = "%1$d accommodations listed!";
     public static final String MESSAGE_FRIENDS_LISTED_OVERVIEW = "%1$d friends listed!";
     public static final String MESSAGE_TRAVELPLANS_LISTED_OVERVIEW = "%1$d travel plans listed!";
+    public static final String MESSAGE_INVALID_STARTANDENDDATE = "Start Date should be before or "
```
Comment (lyeyixian) [inline src/main/java/seedu/address/commons/core/Messages.java:24]: I think its better to name it as ....START_AND_END_DATE?

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -114,6 +114,10 @@ public AddAccommodationCommand parseAccommodation(String args) throws ParseExcep
         WanderlustDate startDate = ParserUtil.parseDate(argMultimap.getValue(PREFIX_START).get());
         WanderlustDate endDate = ParserUtil.parseDate(argMultimap.getValue(PREFIX_END).get());
 
+        if (startDate.getValue().compareTo(endDate.getValue()) > 0) {
+            throw new ParseException("end date should not be earlier than start date");
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:118]: Perhaps you can use the MESSAGE_INVALID_START_AND_END_DATE here?

**Review by lyeyixian — APPROVED** [review]
LGTM. Just some small changes needed

**Review by jiaweiteo — APPROVED** [review]
LGTM

**Pull request merged**